### PR TITLE
Add catalog-info.yaml for dev portal

### PR DESCRIPTION
**Pull request by rjgildea**

### CODE REVIEW

File: catalog-info.yaml [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+    diamond.ac.uk/viewdocs-url: https://zocalo.readthedocs.io
+  tags:
+    - python
+spec:
+  type: library
+  lifecycle: production
+  owner: group:data-analysis
+dependsOn:
+  - resource:zocalo-rabbitmq
+  - resource:workflows
```
Comment (garryod) [inline catalog-info.yaml:20]: What is this resource? Am I right to assume it has not yet been created?

Comment (rjgildea) [reply catalog-info.yaml:20]: This resource would be created by python-workflows#167 - sorry I should have linked to that PR here

Comment (garryod) [reply catalog-info.yaml:20]: Gotcha, must've missed that one in the deluge I've been getting this morning :laughing:

Comment (garryod) [reply catalog-info.yaml:20]: `workflows` appears to be a `Component`, not a `resource`, I expect this needs changed

**Review by garryod — APPROVED** [review]
LGTM

**Pull request merged**